.rubocop.yml

```
@@ -44,3 +43,235 @@ Style/TrailingCommaInHashLiteral:
   Enabled: true
   EnforcedStyleForMultiline: consistent_comma
 
+Gemspec/DeprecatedAttributeAssignment: # new in 1.30
+  Enabled: true
+Gemspec/DevelopmentDependencies: # new in 1.44
+  Enabled: true
+Gemspec/RequireMFA: # new in 1.23
+  Enabled: true
```

That will make releasing via CI pretty hard. Here is an example how I was hacking it:
https://github.com/anycable/graphql-anycable/commit/8803f46b0fa6e1827914fc9e49877b9e72ec6818#diff-4d14704b6b88fb06db888f96c03a8e9b3a5e07a4ee566d97d4111b2c05210e84L64

Would like to disable:
```suggestion
Gemspec/RequireMFA: # new in 1.23
  Enabled: false
```